pvfactors/engine.py

```diff
@@ -43,12 +41,10 @@ def __init__(self, params, vf_calculator=VFCalculator(),
             for the back surface of the selected pvrow (Default = None)
 
         """
-        self.params = params
         self.vf_calculator = vf_calculator
         self.irradiance = irradiance_model
-        self.cls_pvarray = cls_pvarray
-        self.is_fast_mode = isinstance(fast_mode_pvrow_index, int) \
-            and fast_mode_pvrow_index < params['n_pvrows']
```

Comment:
since ``params`` is not passed anymore to the engine, we can't do this check for the user anymore